<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-BG0G3RF" failures="1" tests="2" name="Default test(failed)(failed)" time="0.053" errors="0" timestamp="2024-11-14T15:36:05 ICT">
  <testcase classname="testNgPrioritize.TestNgRetry" name="test2" time="0.006"/>
  <testcase classname="testNgPrioritize.TestNgRetry" name="test3" time="0.009">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNgPrioritize.TestNgRetry.test3(TestNgRetry.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="testNgPrioritize.TestNgRetry" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed)(failed) -->
